ingest-app/src/cmr/ingest/validation/validation.clj

```diff
@@ -88,9 +88,13 @@
   [concept]
   (if-errors-throw :bad-request
                    (if (mt/umm-json? (:format concept))
-                     (umm-spec/validate-metadata (:concept-type concept)
-                                                 (:format concept)
-                                                 (:metadata concept))
+                     (let [umm-version (mt/version-of (:format concept))
+                           accept-version (config/ingest-accept-umm-version)]
+                       (if (>= 0 (compare umm-version accept-version))
+                         (umm-spec/validate-metadata (:concept-type concept)
+                                                     (:format concept)
+                                                     (:metadata concept))
+                         [(str "umm version is higher than ingest-accept-umm-version" (pr-str accept-version))]))
```

Comment:
1. We don't know which version is in development, it might not exist, current-version might be official too. We only know which version is ingest accepted. 
This message has to be shared by both development and the operation environment, so we can't even tell through the comparison of the current-version and the ingest-accept-version.

2. If it's higher than the accepted version, we don't need to validate that their version is valid, just reject and tell them they can only ingest versions equal or below the accept-version.

3. If it's lower than the accepted version, we will allow them to ingest and then further down the code, it will catch it if it's an invalid umm version.